src/App.js

```diff
@@ -45,99 +47,71 @@ class App extends Component {
-          </Row>
-        </Container>
-        <footer>
-          <div className="footer-top">
-            <Container>
+          {web3Provider && web3Provider.isMetaMask && !web3Provider.isNiftyWallet && (
+            <>
               <Row>
-                <div className="col-lg-12">
-                  <img src="assets/img/powered_by_iov.svg" className="img-fluid powered_by" alt="powered_by" />
-                </div>
-                <div className="col-lg-3">
-                  <span className="footer-title mb-3">What is RIF?</span>
-                  <p className="mb-5">RIF goal is to enable Decentralized Sharing Economies to flourish in order to empower and protect the value of individuals.</p>
-                </div>
-                <div className="col-lg-2">
-                  <b><a href="https://www.rsk.co/" target="_blank" rel='noopener noreferrer'>RSK</a></b>
-                  <a href='https://developers.rsk.co' target='_blank' alt='libs' rel='noopener noreferrer'>Documentation</a>
-                  <a href='https://faucet.testnet.rsk.co/' target='_blank' alt='libs' rel='noopener noreferrer'>tRBTC Faucet</a>
-                </div>
-                <div className="col-lg-2">
-                  <b><a href="https://www.rifos.org/" target="_blank" rel='noopener noreferrer'>RIF</a></b>
-                  <a href='https://developers.rsk.co/rif' target='_blank' alt='rif_whitepaper' rel='noopener noreferrer'>Documentation</a>
-                  <a href='https://www.rifos.org/assets/whitepapers/rif-whitepaper-en.pdf' target='_blank' alt='rif_whitepaper' rel='noopener noreferrer'>Whitepaper</a>
-                </div>
-                <div className="col-lg-2">
-                  <b><a href="https://www.rifos.org/rif-name-service/" target="_blank" rel='noopener noreferrer'>RIF Name Service</a></b>
-                  <a href='https://developers.rsk.co/rif/rns' target='_blank' alt='libs' rel='noopener noreferrer'>Documentation</a>
-                  <a href='https://manager.rns.rifos.org/' target='_blank' alt='libs' rel='noopener noreferrer'>Manager</a>
-                </div>
-                <div className="col-lg-3">
-                  <b><a href="https://www.rifos.org/rif-lumino-network/" target="_blank" rel='noopener noreferrer'>RIF Lumino Network</a></b>
-                  <a href='https://explorer.lumino.rifos.org/' target='_blank' alt='rif_whitepaper' rel='noopener noreferrer'>Explorer</a>
-                  <a href='https://developers.rsk.co/rif/lumino' target='_blank' alt='rif_whitepaper' rel='noopener noreferrer'>Docs</a>
-                </div>
+                <Col>
+                  <AddRIFTokenComponent />
+                </Col>
               </Row>
-            </Container>
-          </div>
-        </footer>
+              <Row style={{ marginTop: '2em' }}>
+                <Col>
+                  <TextInput name='your_address' label='Your Address' value={account ?? ''} inputColorVariant='secondary' />
```

Comment:
This should be editable in case I want to send to a different address.

Reply:
Good to go